Userland/Libraries/LibWeb/Layout/LayoutState.h

```diff
@@ -117,6 +117,9 @@ struct LayoutState {
         void add_floating_descendant(Box const& box) { m_floating_descendants.set(&box); }
         auto const& floating_descendants() const { return m_floating_descendants; }
 
+        void set_style_overrides(Painting::BordersData const& style_overrides) { m_style_overrides = style_overrides; };
+        auto const& style_overrides() const { return m_style_overrides; }
```

Comment:
Let's call this something like `set_override_borders_data()` to match the new type.

Reply:
Thanks, done.